docs/admin/cluster-management.md

```diff
@@ -66,37 +66,43 @@ Instance Group will take care of putting appropriate image on new machines and s
 In other environments you may need to configure the machine yourself and tell the Kubelet on which machine API server is running.
 
 
-### Horizontal auto-scaling of nodes (GCE)
+### Cluster autoscaling
 
-If you are using GCE, you can configure your cluster so that the number of nodes will be automatically scaled based on:
+If you are using GCE or GKE, you can configure your cluster so that it is automatically rescaled based on
+pod needs.
 
- * CPU and memory utilization.
- * Amount of of CPU and memory requested by the pods (called also reservation).
+As desribed in [Compute Resource](../user-guide/compute-resources.md), user can declare how much
+CPU and memory a pod needs to run. This amount is called request and is used by Kubernetes scheduler (along with ports
+and labels, if provided) to find a place to start the pod. If there is no node that has enough free capacity or
+doesn't match other pod requirements then the pod has to wait, possibly quite long,
+until some pods are terminated. Or until a new node is added. 
 
-Before setting up the cluster by `kube-up.sh`, you can set `KUBE_ENABLE_NODE_AUTOSCALER` environment variable to `true` and export it.
-The script will create an autoscaler for the instance group managing your nodes.
+Cluster autoscaler looks for the pods that cannot be scheduled and checks if adding a new node, similar
+to the other in the cluster, would help. If yes, then it resizes the cluster to accomodate the waiting pods.
 
-The autoscaler will try to maintain the average CPU/memory utilization and reservation of nodes within the cluster close to the target value.
-The target value can be configured by `KUBE_TARGET_NODE_UTILIZATION` environment variable (default: 0.7) for ``kube-up.sh`` when creating the cluster.
-Node utilization is the total node's CPU/memory usage (OS + k8s + user load) divided by the node's capacity.
-Node reservation is the total CPU/memory requested by pods that are running on the node divided by the node's capacity.
-If the desired numbers of nodes in the cluster resulting from CPU/memory utilization/reservation are different,
-the autoscaler will choose the bigger number. The number of nodes in the cluster set by the autoscaler will be limited from `KUBE_AUTOSCALER_MIN_NODES` (default: 1)
-to `KUBE_AUTOSCALER_MAX_NODES` (default: the initial number of nodes in the cluster).
+Cluster autoscaler also scales down the cluster if it notices that some node is not needed anymore in a sence
+that:
 
-The autoscaler is implemented as a Compute Engine Autoscaler.
-The initial values of the autoscaler parameters set by `kube-up.sh` and some more advanced options can be tweaked on
-`Compute > Compute Engine > Instance groups > your group > Edit group`[Google Cloud Console page](https://console.developers.google.com)
-or using gcloud CLI:
+* It has more that 50% of cpu and memory not requested by any pod.
```

Comment:
I disagree - we can always mark that this strategy may change in the future. 
I wouldn't trust such autoscaler that sometimes may delete some nodes based on some vague conditions.